.changelog/unreleased/breaking-changes/1176-blockindexer-pruning.md

```diff
@@ -0,0 +1,2 @@
+- `[state/indexer/block]` BlockIndexer now has additional method `Prune(retainHeight int64) (int64, int64, error)` ([\#1176](https://github.com/cometbft/cometbft/pull/1176))
+- `[state/txindex]` TxIndexer now has additional method `Prune(retainHeight int64) (int64, int64, error)` ([\#1176](https://github.com/cometbft/cometbft/pull/1176))
```

Comment:
Can you also mention the other to added methods? No need to mention their entire signature

Reply:
Done